custom_components/ltss/__init__.py

```diff
@@ -48,14 +48,20 @@
 
 CONF_DB_URL = "db_url"
 CONF_CHUNK_TIME_INTERVAL = "chunk_time_interval"
-
-CONNECT_RETRY_WAIT = 3
+CONF_DB_RETRY_WAIT = "db_retry_wait"
+CONF_DB_RETRY_LIMIT = "db_retry_limit"
 
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: INCLUDE_EXCLUDE_BASE_FILTER_SCHEMA.extend(
             {
                 vol.Required(CONF_DB_URL): cv.string,
+                vol.Optional(
+                    CONF_DB_RETRY_WAIT, default=3.0
+                ): cv.positive_float,  # time to wait between db reconnects, default 3.0s
+                vol.Optional(CONF_DB_RETRY_LIMIT, default=None): vol.Any(
+                    None, cv.positive_int
+                ),  # max. number of retries when connecting to the DB, no limit by default
```

Comment:
Please keep the previous hard-coded value as the default (i.e. 10), to avoid any confusion when upgrading existing installations.

Reply:
Ok.